dashboard/src/shared/types.ts

```diff
@@ -95,6 +104,7 @@ export interface IChartAttributes {
 
 export interface IChartState {
   isFetching: boolean;
+  hasFinished: boolean;
```

Comment:
nitpipck: `hasFinished` could mean anything, a better name would be `finishedFetching`